specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-06-01/location.json

```diff
@@ -0,0 +1,206 @@
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceTypeParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/OrchestratorVersionProfileListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Container Service Orchestrators": {
+            "$ref": "./examples/ContainerServiceListOrchestrators.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "OrchestratorProfile": {
+      "properties": {
+        "orchestratorType": {
+          "type": "string",
+          "description": "Orchestrator type."
+        },
+        "orchestratorVersion": {
+          "type": "string",
+          "description": "Orchestrator version (major, minor, patch)."
+        },
+        "isPreview": {
+          "type": "boolean",
+          "description": "Whether Kubernetes version is currently in preview."
+        }
+      },
+      "required": [
+        "orchestratorVersion"
+      ],
+      "description": "Contains information about orchestrator."
+    },
+    "OrchestratorVersionProfile": {
+      "properties": {
+        "orchestratorType": {
+          "type": "string",
+          "description": "Orchestrator type."
+        },
+        "orchestratorVersion": {
+          "type": "string",
+          "description": "Orchestrator version (major, minor, patch)."
+        },
+        "default": {
+          "type": "boolean",
+          "description": "Installed by default if version is not specified."
+        },
+        "isPreview": {
+          "type": "boolean",
+          "description": "Whether Kubernetes version is currently in preview."
+        },
+        "upgrades": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OrchestratorProfile"
+          },
+          "description": "The list of available upgrade versions."
+        }
+      },
+      "required": [
+        "orchestratorVersion",
+        "orchestratorType"
+      ],
+      "description": "The profile of an orchestrator and its available versions."
+    },
+    "OrchestratorVersionProfileProperties": {
+      "properties": {
+        "orchestrators": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OrchestratorVersionProfile"
+          },
+          "description": "List of orchestrator version profiles."
+        }
+      },
+      "required": [
+        "orchestrators"
+      ],
+      "description": "The properties of an orchestrator version profile."
+    },
+    "OrchestratorVersionProfileListResult": {
```

Comment:
The type name is a bit long, how about OrchestratorListResult?  Matches with the (new) operationId as well.